kornia/augmentation/_2d/intensity/planckian_jitter.py

```diff
@@ -0,0 +1,118 @@
+                                               [0.3632, 0.4655, 0.6756],
+                                               [0.3586, 0.4655, 0.6820],
+                                               [0.3544, 0.4654, 0.6878],
+                                               [0.3503, 0.4653, 0.6933]]),
+
+                    'CIED': torch.Tensor([[0.5829, 0.4421, 0.2288],
+                                          [0.5510, 0.4514, 0.2948],
+                                          [0.5246, 0.4576, 0.3488],
+                                          [0.5021, 0.4618, 0.3941],
+                                          [0.4826, 0.4646, 0.4325],
+                                          [0.4654, 0.4667, 0.4654],
+                                          [0.4502, 0.4681, 0.4938],
+                                          [0.4364, 0.4692, 0.5186],
+                                          [0.4240, 0.4700, 0.5403],
+                                          [0.4127, 0.4705, 0.5594],
+                                          [0.4023, 0.4709, 0.5763],
+                                          [0.3928, 0.4713, 0.5914],
+                                          [0.3839, 0.4715, 0.6049],
+                                          [0.3757, 0.4716, 0.6171],
+                                          [0.3681, 0.4717, 0.6281],
+                                          [0.3609, 0.4718, 0.6380],
+                                          [0.3543, 0.4719, 0.6472],
+                                          [0.3480, 0.4719, 0.6555],
+                                          [0.3421, 0.4719, 0.6631],
+                                          [0.3365, 0.4719, 0.6702],
+                                          [0.3313, 0.4719, 0.6766],
+                                          [0.3263, 0.4719, 0.6826],
+                                          [0.3217, 0.4719, 0.6882]])
+                    }
+
+
+class PlanckianJitter(IntensityAugmentationBase2D):
+    r"""Apply planckian jitter transformation to input tensor
+    This is physics based color augmentation, that creates realistic
+    variations in chromaticity, this can simulate the illumination
+    changes in the scene. This is based on https://arxiv.org/pdf/2202.07993.pdf
+
+    Args:
+        mode: 'blackbody' or 'CIED'.
+        same_on_batch: apply the same transformation across the batch.
+        p: probability that the random erasing operation will be performed.
+        keepdim: whether to keep the output shape the same as input (True) or broadcast it
+                                to the batch form (False).
+    Shape:
+    - Input: :math:`(C, H, W)` or :math:`(B, C, H, W)`
+    - Output: :math:`(B, C, H, W)`
+
+    Note:
+        Input tensor must be float and normalized into [0, 1]
+
+    Examples:
+        >>> rng = torch.manual_seed(0)
+        >>> input = torch.randn(1, 3, 32, 32)
+        >>> aug = PlanckianJitter(mode='CIED', p=1.0)
+        >>> aug(input)
```

Comment:
Need to add results. Other doctest will fail. Run doctest with `pytest --doctest-modules ./SOMEFILE.py`